literature.bib

```diff
@@ -4481,6 +4609,31 @@ @article{DBW:SCP18
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2017 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+%%% formerly known as RPK+:PLMR17
+@inproceedings{RPK+:ICML17,
+	title = {{On the Expressive Power of Deep Neural Networks}},
+	author = {Maithra Raghu and Ben Poole and Jon Kleinberg and Surya Ganguli and Jascha Sohl-Dickstein},
+	booktitle = ICML,
+	pages = {2847--2854},
+	year = 2017,
+	volume = {70},
+	month = AUG,
+	publisher = {PMLR},
```

Comment:
Why is the publisher in brackets? Isn't this an abbreviation?

Reply:
Thats indeed the publisher name and it is not a variable because it is the only occurrence so far. The publishing model behind this paper is indeed a bit weird but I just took what their website wanted from me.

Reply:
Ok, thank you for clarifying